npm/webpack-dev-server/package.json

```diff
@@ -44,5 +44,15 @@
   },
   "files": [
     "dist"
-  ]
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cypress-io/cypress.git"
```

Comment:
oh, @ryanthemanuel i bet this is why we were getting "authenticity cannot be established" - we were using git+ssh, and having `repository.url` set can force npm to use `https` instead, avoiding `known_hosts` verification. kudos @ZachJW34 for actually tracking this down

Reply:
although i don't think you were touching these files when you first ran into it :confused:

Reply:
Glad I could (unknowingly) help 😅